src/diffpy/utils/tools.py

```diff
@@ -131,3 +133,25 @@ def get_package_info(package_names, metadata=None):
         pkg_info.update({package: importlib.metadata.version(package)})
     metadata["package_info"] = pkg_info
     return metadata
+
+
+def compute_mu_using_xraydb(sample, energy, density=None):
```

Comment:
let's change `sample` to `composition` or maybe `sample_composition`.

I think we probably  need `packing_fraction` as an optional arg.